plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -70,7 +71,21 @@ const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
-  const { about, loading, error } = useInfo();
+  const { about: initialAbout, loading, error } = useInfo();
+  const { lastSignal } = useSignal<DevToolsInfo>('devtools:info');
+  const [about, setAbout] = React.useState<DevToolsInfo | undefined>(
+    initialAbout,
+  );
+
+  useEffect(() => {
+    setAbout(initialAbout);
+  }, [initialAbout]);
+
+  useEffect(() => {
+    if (lastSignal) {
+      setAbout(lastSignal);
+    }
+  }, [lastSignal]);
```

Comment:
As I understand it `useInfo` & `useSignal` 'devtools:info' both have the power to update "about". If `lastSignal`  is not yet set `initalAbout` is the default. Couldn't this be simply replaced by smth. like:

```ts
const about = lastSignal ?? initialAbout;
```

This would not cover the case that `useInfo` updates the `initalAbout` - but does that ever update & should it be prioritized? We might want to reduce the renders here one way or another as it would render twice every second due to the signal hook + effect.

Reply:
Yeah that may work, will change!